src/functions/template-tags/loop.php

```diff
@@ -128,7 +128,13 @@ function tribe_get_events_title( $depth = true ) {
 
 		$tribe_ecp = Tribe__Events__Main::instance();
 
-		$title = sprintf( esc_html__( 'Upcoming %s', 'the-events-calendar' ), $events_label_plural );
+		if ( is_single() && tribe_is_event() ) {
```

Comment:
Is the `tribe_is_event()` check necessary? If it happens to _not_ be an event, then failing over to "Upcoming %s" would be a little funky.